### PR TITLE
[MOV] im_livechat: move document_viewer out of public_livechat.js

### PR DESCRIPTION
**Pull request by Nemael**
Task-2212347

### CODE REVIEW

File: addons/im_livechat/static/src/legacy/public_livechat.js [hunk 1]
```diff
@@ -3535,399 +3535,3 @@ ThreadWidget.ORDER = ORDER;
...
-        this.dragStartX = this.dragstopX = 0;
-        this.dragStartY = this.dragstopY = 0;
-    },
-    /**
-     * Render the active attachment
-     *
-     * @private
-     */
-    _updateContent: function () {
-        this.$('.o_viewer_content').html(QWeb.render('im_livechat.legacy.mail.DocumentViewer.Content', {
```
Comment (alexkuhn) [inline addons/im_livechat/static/src/legacy/public_livechat.js:3670]: ci/security due to this move

**Review by alexkuhn — APPROVED** [review]
@seb-odoo ci/security to check (moved LOCs)

**Comment by seb-odoo** [comment]
@robodoo override=ci/security
Legacy code move still requiring to manually set html. Content is safe as it is coming from a template.

**Comment by alexkuhn** [comment]
@robodoo r+

**Pull request closed without merging**